FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj

```diff
@@ -213,7 +213,6 @@
 				A24E011B17F9A42A001252DB /* Frameworks */,
 				A24E011C17F9A42A001252DB /* Resources */,
 				279D0DCE5C316DE54BD5FF49 /* [CP] Embed Pods Frameworks */,
-				F04559B5D97B33B48486C223 /* [CP] Copy Pods Resources */,
```

Comment:
believe most of these changes are due to cocoapods version update + updating minimum deployment target but let me know if needs cleanup